onnxscript/rewriter/_matcher.py

```diff
@@ -352,6 +362,7 @@ def match(
         complications which require careful consideration.
         """
         self._tracer = tracer
+        self._graph_or_function = graph_or_function[0].graph
```

Comment:
@justinchuby : this was a quick hack to test this implementation. Would be nice if `ir.Function` exposed a `graph()` property. Basically this should be `graph_or_function if isinstance(graph_or_function, ir.Graph) else graph_or_function._graph`.

Reply:
Oh I see. We need it for testing. Let me think about this

Reply:
https://github.com/onnx/ir-py/pull/158